mtc.go

```diff
@@ -53,6 +54,24 @@ type Claims struct {
 	Unknown     []UnknownClaim
 }
 
+type EvidenceType uint16
+
+const (
+	EmptyEvidenceType EvidenceType = iota
+	X509ChainEvidenceType
+)
+
+type Evidence struct {
```

Comment:
If this is only for the shadow certs, then we don't need polymorphism: just have a list of certs or an empty list.

If we want to reuse this for something else, then we want a list of possible evidence, instead of just one piece of evidence.